src/features/content-insights/__tests__/ContentInsightsSummary.test.tsx

```diff
@@ -17,11 +17,14 @@ const mockData = [
     { start: 7, previewsCount: 1, type: 'day' },
 ] as GraphData;
 
+const mockError = jest.fn();
```

Comment:
Does this need to be a mock? Can we pass the `error` as appropriate into the `getWrapper`?

Reply:
We would have to mock an axios error with different fields, can't pass in a regular error

Reply:
See my comment about the error type, that should likely simplify the typing situation